cms/djangoapps/contentstore/views/course.py

```diff
@@ -549,13 +555,21 @@ def format_library_for_view(library):
     active_courses, archived_courses = _process_courses_list(courses_iter, in_process_course_actions, split_archived)
     in_process_course_actions = [format_in_process_course_view(uca) for uca in in_process_course_actions]
 
+    redirect_to_library_authoring_mfe = (
+        ENABLE_LIBRARY_AUTHORING_MICROFRONTEND and
+        LIBRARY_AUTHORING_MICROFRONTEND_URL and
+        REDIRECT_TO_LIBRARY_AUTHORING_MICROFRONTEND.is_enabled()
+    )
+
```

Comment:
Non-blocking suggestion: Consider exposing a `should_redirect_to_library_authoring_mfe()` function from `library.py` (or really anywhere else sensible) to consolidate this logic. I find that having a single source-of-truth for whether something is On or Off can help avoid moments during rollout like *"Wait, why are we only redirecting to the new feature in half of the places? Oh right, it's because one place checks the Django setting and the flag, where as the other place only checks the flag and whether the URL is set"*, etc.

Reply:
Good call.  Done.